gremlin-python/src/main/python/examples/basic_gremlin.py

```diff
@@ -25,9 +26,13 @@
 
 
 def main():
-    rc = DriverRemoteConnection('ws://localhost:8182/gremlin', 'g')
+    server_url = os.getenv('GREMLIN_SERVER_URL', 'ws://localhost:8182/gremlin').format(45940)
```

Comment:
Nit: add comment with context where this weird port is coming from
```suggestion
    # if there is a port placeholder in the env var then we are running with docker so set appropriate port 
    server_url = os.getenv('GREMLIN_SERVER_URL', 'ws://localhost:8182/gremlin').format(45940)
```